bigtable/google/cloud/bigtable/table.py

```diff
@@ -276,6 +284,31 @@ def read_rows(self, start_key=None, end_key=None, limit=None,
         # We expect an iterator of `data_messages_v2_pb2.ReadRowsResponse`
         return PartialRowsData(response_iterator)
 
+    def mutate_rows(self, rows):
+        """Mutates multiple rows in bulk.
+    
+        :type rows: list
+        :param rows: List or other iterable of :class:`.DirectRow` instances.
+        
+        :rtype: list
+        :returns: A list of tuples (``MutateRowsResponse.Entry`` protobuf 
+                  corresponding to the errors, :class:`.DirectRow`) 
+        """
+        _check_rows(self.name, rows)
+        mutate_rows_request = _mutate_rows_request(self.name, rows)
+
+        unsuccessfully_mutated_rows = []
+        client = self._instance._client
+        responses = client._data_stub.MutateRows(mutate_rows_request)
+        for response in responses:
+            for entry in response.entries:
+                if not entry.status.code:
```

Comment:
So each [`Entry`][1] has a `Status` and [`Status.code`][2] is an `int32`. Can you change this check instead to:

```python
# 0 is empty value for int32, indicates code is unset
if entry.status.code == 0:
    ...
```

[1]: https://github.com/googleapis/googleapis/blob/e0922c4ea8e2f03c11f5dce405b79ecdf110792b/google/bigtable/v2/bigtable.proto#L255
[2]: https://github.com/googleapis/googleapis/blob/e0922c4ea8e2f03c11f5dce405b79ecdf110792b/google/rpc/status.proto#L82